kivy/core/window/_window_sdl2.pyx

```diff
@@ -320,6 +321,9 @@ cdef class _WindowSDL2Storage:
         cdef void * p = SDL_GL_GetProcAddress(c_name)
         return <intptr_t>p
 
+    def reset_gl_context(self):
```

Comment:
Do we need this function? 
Why not call the function directly instead of making it like this

Reply:
Do you mean the `SDL_GL_GetProcAddress `function or `reset_gl_context`?
Anyway `SDL_GL_GetProcAddress` returns void pointer so there's no point to call it directly because we need to cast it to intptr_t, and about `reset_gl_context ` I don't know how to call it directly because it can't be imported from `kivy.graphics.instructions` from python